src/main/java/com/google/edith/servlets/Item.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+ /** Used to hold the properties of an item Entity in datastore. */
+public final class Item {
+  double price; 
+  long quantity;  
+//   String Name; 
```

Comment:
Can we remove the commented part ?

Reply:
The reason I have this commented part is because these are theses are the fields we chose in the data model. They just weren't immediately important for this PR, so I left the commented out.